lib/spawn/cluster/node/server.ex

```diff
@@ -11,14 +11,14 @@ defmodule Spawn.Cluster.Node.Server do
 
   def request(%{topic: topic, body: body, reply_to: reply_to} = req)
       when is_binary(body) do
-    Logger.debug("Received Actor Invocation via Nats on #{topic}")
+    Logger.info("Received Actor Invocation via Nats on #{topic}")
     headers = Map.get(req, :headers, [])
 
     handle_request(topic, body, reply_to, headers)
   end
 
   def request(%{topic: topic, body: _, reply_to: _reply_to} = _req) do
-    Logger.debug("Received Invalid Actor Invocation via Nats on #{topic}")
+    Logger.info("Received Invalid Actor Invocation via Nats on #{topic}")
```

Comment:
Wdyt for using warning instead info or debug ?

Reply:
warning looks better!